src/patternfly/components/TreeView/tree-view-node.hbs

```diff
@@ -1,21 +1,32 @@
 {{#> tree-view-content}}
-  <{{# if tree-view-node--type}}{{tree-view-node--type}}{{else}}button{{/if}} class="pf-c-tree-view__node{{#if tree-view-node--modifier}} {{tree-view-node--modifier}}{{/if}}{{#if tree-view-node--IsActive}} pf-m-active{{/if}}"
+  <{{# if tree-view-node--type}}{{tree-view-node--type}}{{else}}button{{/if}} class="pf-c-tree-view__node{{#if tree-view-node--modifier}} {{tree-view-node--modifier}}{{/if}}"
     {{#if tree-view-node--attribute}}
       {{{tree-view-node--attribute}}}
     {{/if}}>
     {{#if tree-view-node--IsToggle}}
       {{> tree-view-node-toggle-icon}}
     {{/if}}
     {{#if tree-view-node--HasCheckbox}}
-      {{> tree-view-node-check}}
+      {{> tree-view-node-toggle}}
     {{/if}}
     {{#if tree-view-node--HasFolderIcon}}
       {{> tree-view-node-icon tree-view-node-icon--IsFolder="true"}}
     {{/if}}
-    {{#if tree-view-node--text}}
-      {{#> tree-view-node-text}}
-        {{{tree-view-node--text}}}
-      {{/tree-view-node-text}}
+    {{#if tree-view-node--HasCheckbox}}
+      {{#> tree-view-node-main}}
+        {{> tree-view-node-check}}
```

Comment:
```suggestion
        {{> tree-view-node-check tree-view-node-check--IsChecked=tree-view-node--checkbox--IsChecked}}
```

Reply:
I think this is meant to be `{{> tree-view-node-check tree-view-node-check--IsChecked="true"}}` ?